### ISSUE TITLE
migrate from `torequests` to `httpx`.

### ISSUE THREAD

**Issue by T-256** [ClericPy/ichrome#148]
less dependencies and lighter weight

**Comment by ClericPy** [comment]
Indeed, I had planned to remove torequests from the ichrome library two years ago. The reason why I chose aiohttp at the beginning was that its concurrency capability was an order of magnitude higher than other libraries (httpx and requests are much slower). On the one hand, aiohttp itself comes with a Websocket coroutine client. When ichrome was first created, there were few options with such matching functions. During the recent period, I first implemented a morebuiltins library to replace various functions in torequests.utils, and then the next step was to start httpx to replace the http and websocket parts of aiohttp. I was busy at work and could only write code a little bit in my spare time. If you are looking forward to a more professional library, you can consider the Playwright series of Microsoft Team. One of the reasons for the creation of ichrome was that there were no options at the time, so it was developed on a temporary basis. Relatively speaking, the test coverage is not as good as most other libraries currently

**Comment by ClericPy** [comment]
torequests has been removed
